### ISSUE TITLE
demos/serverless: example is incompatible with Windows 10 Command Prompt; not documented

### ISSUE THREAD

**Issue by swoldemi** [aws/amazon-chime-sdk-js#277]
Example in question lives here: https://github.com/aws/amazon-chime-sdk-js/tree/master/demos/serverless

**Important:** Please attach INFO-level logs with timestamps to the report from *before* the session started up until the point at which the unexpected behavior occured. For multi-participant calls, attach a log file for each participant if possible.
```
C:\Users\me\Projects\amazon-chime-sdk-js\demos\serverless>  node ./deploy.js -r us-east-1 -b my-bucket -s testing-stack -a meeting
,aws-cli/1.16.198 Python/3.6.0 Windows/10 botocore/1.12.188
,
,SAM CLI, version 0.47.0
,
Verifying application meeting
Application ../browser/dist/meeting.html does not exist. Rebuilding demo apps
Command npm failed with ENOENT <---- This shouldn't happen
C:\Users\me\Projects\amazon-chime-sdk-js\demos\serverless> where npm
C:\Program Files\nodejs\npm
C:\Program Files\nodejs\npm.cmd
```

**Meeting and attendee IDs:**
N/A

**Timeline**
Outline the main actions the user performed and when:
See Command Prompt output above.

**Describe the bug**
Describe the bug and the timestamp at which it happened relative to the timestamp in the log file.
See Command Prompt output above.
`deploy.js` is not considerate of Windows Command Prompt users. For [`child_process.spawnSync`](https://nodejs.org/api/child_process.html#child_process_child_process_spawnsync_command_args_options) to work on Windows, it seems some additional parameters should be passed. This happens transiently across different commands (that DO exist in PATH), but is sometimes solved by appending the `.cmd` suffix on the command. (e.g. `npm.cmd` will not result in ENOENT). In any case, running `node ./deploy.js -r us-east-1 -b mybucket -s stackname -a appname` to deploy the application will still fail because `cp` is not a native Command Prompt resource: https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/serverless/deploy.js#L137-L142

**Screenshots**
If applicable, add screenshots to help explain the problem.
N/A

**Platform**
 - OS :Microsoft Windows 10 [Version 10.0.18362.720]
 - Browser:  N/A
 - Browser Version: N/A 
 - Amazon Chime SDK GitHub Commit: f8c345b95472cc2aea26e7e399ca72f3523900d2

As a preface, yes a customer can use WSL or Git Bash, but this isn't documented either.

**Comment by waisimon** [comment]
same problem with me, and fail step for `cp` command in windows platform

- for `ENOENT` error, will work if add `shell: true` at the `options` parameter in `spawnSync` function
   > spawnSync(command, args, { ...options, shell: true}

**Comment by ltrung** [comment]
This should have been fixed in #307 so closing.
